Documentation updates (#502)
grafana/grizzly: docs/content/grafana.md

A folder simply has a name and a title.

## Nested Folders

> Grizzly supports creation of nested folders, which can be useful when you don't want to have ALL your dashboards in a single one.
> To achieve that, you need to improve your DashboardFolder definition to add the uuid of the parent folder:
> 
> ```
> apiVersion: grizzly.grafana.com/v1alpha1
> kind: DashboardFolder
> metadata:
>   name: defaultnestedfolder
> spec:
>   parentUid: default
>   title: nestedfolder
>   uid: nesteddefaultfolder
> ```
> 
> This will create a new folder called `nestedfolder`with a uid: `nesteddefaultfolder`under the folder: `default`in your grafana instance.

Reviewer: ```suggestion
Grizzly supports the creation of nested folders.
To create one, specify the *uid* of the parent folder to a `DashboardFolder` definition:

```yaml
apiVersion: grizzly.grafana.com/v1alpha1
kind: DashboardFolder
metadata:
  name: nestedFolderSample
spec:
  parentUid: sample
  title: Nested folder
```

This will create a new folder "Nested folder" under the `sample` folder.
```

To keep the code sample linked to what the documentation was previously describing :)

Reviewer: Looks like github doesn't like edit suggestions with nested backticks blocks :(
